README.md

```diff
@@ -686,6 +686,8 @@ This setting is stored in `job.options.backoff` as `{strategy, delay}`.
 
 Defaults to `'immediate'`.
 
+###### _NOTE: You should set `activateDelayedJobs` to `true` from at least one of your queues. See [Queue settings section](#settings)_
```

Comment:
How about we change this language to "N.B.: bee-queue's retries jobs by re-enqueuing failed jobs as delayed jobs. This means you need to enable `activateDelayedJobs` on any queue that uses retry/backoff functionality." and add this to to the `Job#retries(n)` subsection as well.